### ISSUE TITLE
Node ids should always be escaped

### ISSUE THREAD

**Issue by yuhan0** [Macroz/tangle#3]
Keywords are passed unescaped, so eg. the following produces a syntatically invalid dot file, as well as any keyword that contains an illegal character.

```clojure
(tangle/graph->dot
  #{:a->b :b->c}
  {:a->b :b->c}
  {})
```
#### Output:
```
graph {

graph[dpi=100, rankdir=TP]
b->c
a->b

a->b -- b->c
}
```

**Comment by Macroz** [comment]
Thanks for the report. This is definitely a missing feature as of now.

**Comment by Macroz** [comment]
I think this is now fixed in the v0.2.1 release that I just deployed to clojars. Please confirm that it's working for you!
